modules/get-started/pages/licensing/add-license-redpanda/kubernetes.adoc

```diff
@@ -0,0 +1,326 @@
+  --values redpanda-license.yaml --reuse-values
+```
+
+--set::
++
+[,bash]
+----
+helm upgrade --install redpanda redpanda/redpanda \
+  --namespace <namespace> \
+  --create-namespace \
+  --set enterprise.licenseSecretRef.name=redpanda-license \
+  --set enterprise.licenseSecretRef.key=license
+----
+
+====
+--
+======
+
+[[inline]]
+=== Provide the license inline
+
+If you prefer to provide the license string directly, you can do so as follows:
+
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha2
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    enterprise:
+      license: <license-key>
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+
+--
+Helm::
++
+--
+
+[tabs]
+====
+--values::
++
+.`redpanda-license.yaml`
+[source,yaml]
+----
+enterprise:
+  license: <license-key>
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --values redpanda-license.yaml --reuse-values
+```
+
+--set::
++
+[,bash]
+----
+helm upgrade --install redpanda redpanda/redpanda \
+  --namespace <namespace> \
+  --create-namespace \
+  --set enterprise.license=<license-key>
+----
+
+====
+--
+======
+
+If neither the Kubernetes Secret nor the license string is provided, Redpanda looks for the license at `/etc/redpanda/redpanda.license`.
+
+[[console]]
+=== Use Redpanda Console
+
+You can upload a license directly through Redpanda Console.
+
+See xref:console:ui/add-license.adoc[].
+
+== Verify a license
+
+After adding or updating a license, you can use `rpk` to verify that the license was set.
+
+include::get-started:partial$licensing/check-license.adoc[]
+
+== Update an existing license
+
+The process for updating a license depends on how it was originally applied:
+
+- <<secret-update, Update the Kubernetes Secret>>.
+- <<inline-update, Update the license string inline in your Helm values or manifest file>>.
+- xref:console:ui/add-license.adoc[Use Redpanda Console]
+
+When a new license is uploaded, enterprise features in Redpanda Self-Managed are unlocked immediately without requiring a cluster restart. However, to unlock enterprise features in Redpanda Console, you must restart the Redpanda Console instance.
+
+[[secret-update]]
+=== Update the Kubernetes Secret
+
+If the license is provided through a Kubernetes Secret, follow these steps to update it:
+
+. Download the updated license file and overwrite the existing `redpanda.license` file.
+
+. Delete the existing Secret:
++
+[,bash]
+----
+kubectl delete secret redpanda-license --namespace <namespace>
+----
+
+. Create a new Secret with the updated license:
++
+[,bash]
+----
+kubectl create secret generic redpanda-license \
+  --from-file=license=./redpanda.license \
+  --namespace <namespace>
+----
++
+NOTE: The new license automatically takes effect. You do not need to restart the Redpanda brokers. 
```

Comment:
This isn't true in helm AFAIK because the license is set via the post install/upgrade job which requires the user to run `helm upgrade` to re-trigger.

I don't think this is true for the operator either as updating the license in place wouldn't trigger a reconciliation.

Have you observed different?

Reply:
Oh you're right. This is only true if you use Console (or the admin API).